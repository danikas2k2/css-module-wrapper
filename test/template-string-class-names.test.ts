import path from 'path';
import { compile } from './utils/compile';
import { DIR } from './utils/ufs';

it.skip('template string class names', async () => {
    const [output] = await compile({
        entry: path.resolve(DIR, 'template-string-class-names.tsx'),
    });
    expect(output).toMatchSnapshot();
});
