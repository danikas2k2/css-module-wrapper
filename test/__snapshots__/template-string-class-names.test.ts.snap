// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`template string class names 1`] = `
""use strict";
import { jsx } from "react/jsx-runtime";
import ___Button_css_ from "./Button.css";
import __classNames__bind from "classnames/bind.js";
const __classNames = __classNames__bind.bind(___Button_css_);
export function Button({
  color = "neutral",
  variant = "solid",
  size = "small",
  spacing = "default",
  disabled,
  ...props
}) {
  return /* @__PURE__ */ jsx(
    "button",
    {
      className: __classNames(\`Button\`, \`color-\${color}\`, \`variant-\${variant}\`, \`size-\${size}\`, \`spacing-\${spacing}\`),
      disabled,
      "aria-disabled": disabled,
      ...props
    }
  );
}
"
`;
